src/main/scala/redis/commands/Keys.scala

```diff
@@ -91,4 +91,7 @@ trait Keys extends Request {
   def `type`(key: String): Future[String] =
     send(Type(key))
 
+  def scan(cursor: Int = 0, count: Option[Int] = None, matchGlob: Option[String] = None): Future[(Int, Seq[String])] =
```

Comment:
Maybe we could use for Scan Replies
`case class Cursor[T](index:Int, data:Seq[T])`

instead of `(Int, Seq[T])`

(feel free to rename if needed)